build/Dockerfile.build

```diff
@@ -1,10 +1,9 @@
-FROM registry.access.redhat.com/ubi8/ubi-minimal as builder
+FROM registry.access.redhat.com/ubi8/go-toolset AS builder
 
-RUN microdnf install -y make golang; microdnf -y clean all
-
-FROM builder
-
-RUN mkdir -p /output
 COPY . /ocs-operator
 WORKDIR /ocs-operator
-RUN make build-go OUTDIR_BIN=/output
```

Comment:
How do we get the binary out from the container?

Reply:
Why do we need to get it out? It won't be of any use.

If you want it, you'll have to run the container and copy it.
I tried adding volume mount while building, it doesn't work.

Reply:
The entire point of the containerized build is to get the binary to put it into the final operator container image. It doesn't do anything to build a binary in a container and then do nothing with it. So the binary built in the `build/Dockerfile.build` container and use it as the main binary in `build/Dockerfile`.